### ISSUE TITLE
RTX 3070 driver shows as MS driver

### ISSUE THREAD

**Issue by kch789** [jamesstringerparsec/Easy-GPU-P#79]
Hi, 

Using the scirpt I have successfully installed virtual machine (win 11 pro insider on host, 11 pro in client - both polish versions) got resolution of choice via Parsec, but GPU seems to be unavailable. In device manager I've got RTX 3070 with Microsoft driver.
![s](https://user-images.githubusercontent.com/87370859/149497168-de9b1ca4-b6a9-45b9-b11d-2be073cd80ec.png)
For example furmark:
![s2](https://user-images.githubusercontent.com/87370859/149497230-41393ccc-bbef-4614-912d-3ef140164628.png)
Same story with games, they aren't working at all.

I've done GPU-P before using CraftComputing's youtube tutorial (when I was doing it driver folders names were different in DriverStore location), gpu acceleration was working, but some games were not able to start like Wither 3, Forza Horizon 4 and 5, so I wanted to try this method.

Is it possible that this problem is related to polish language version?

**Comment by jamesstringerparsec** [comment]
See the readme regarding OpenGL and Vulkan.  You have the correct driver
installed

**Comment by KChrzanowski789** [comment]
Woah, that was quick response. The thing is, directX games aren't working too. Got some flickering issues with interfaces in progragms as well
.

**Comment by jamesstringerparsec** [comment]
You can try upgrading your computer OS to Win11, and creating a new VM with Win11.  Likely will provide better game support for DX games.

**Comment by jamesstringerparsec** [comment]
Are you using freesync or gsync on your host?

**Comment by KChrzanowski789** [comment]
> You can try upgrading your computer OS to Win11, and creating a new VM with Win11. Likely will provide better game support for DX games.

I specified that I am using win 11 pro on both.
Turned off G-sync compatible on all monitors, no change.

**Comment by KChrzanowski789** [comment]
Already tried, nothing changed.

**Comment by jamesstringerparsec** [comment]
OK, in that case you can try this beta nvidia driver on the host, which is supposedly made for this tech, no idea if it will help though.  https://developer.nvidia.com/cuda/wsl/download

**Comment by KChrzanowski789** [comment]
Unfortunately, same thing.

**Comment by Gravlok** [comment]
I had some issues with games. Did you install all additional stuff like .NET Frameworks 2.0 and 3.0? DirectX Dependency Fix also became useful. 
https://obsproject.com/forum/resources/directx-dependency-fixer-dxfix.87/

**Comment by KChrzanowski789** [comment]
Yes, everything essential is installed. Everything green in dependency fixer.

**Comment by cfelicio** [comment]
Go to Windows/System32 and look for these files:

![image](https://user-images.githubusercontent.com/7227683/159151790-c2b30b27-e6e3-4bb1-b08d-87e911abc7f6.png)

I had to manually copy these from the host to the guest. After doing that and a reboot, I got furmark working. Good luck and let me know if it helps!

**Comment by pedoc** [comment]
> Go to Windows/System32 and look for these files:

I can see the files in the virtual machine and run the nvidia-smi command successfully, but still can't see the graphics card in task manager.

![image](https://user-images.githubusercontent.com/6604230/219592481-37651872-3e48-4131-9736-960386d7c039.png)

![image](https://user-images.githubusercontent.com/6604230/219592519-42405048-bc23-42a5-969f-9ac7954f11ce.png)

![image](https://user-images.githubusercontent.com/6604230/219592647-f1a423de-2e39-44f5-ae98-c9603d16cb97.png)
